gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -46,17 +47,19 @@ public String getLastActiveProjectId() {
 
   /** Set the id of the last project successfully activated by the user. */
   public void setLastActiveProjectId(@NonNull String id) {
-    preferences
-      .edit()
-      .putString(ACTIVE_PROJECT_ID_KEY, id)
-      .apply();
+    preferences.edit().putString(ACTIVE_PROJECT_ID_KEY, id).apply();
   }
 
   /** Removes the last active project id in the local value store. */
   public void clearLastActiveProjectId() {
-    preferences
-      .edit()
-      .remove(ACTIVE_PROJECT_ID_KEY)
-      .apply();
+    preferences.edit().remove(ACTIVE_PROJECT_ID_KEY).apply();
+  }
+
+  public boolean shouldUploadMediaOverWifiOnly() {
```

Comment:
We might want to change this to `shouldUploadMediaOverUnmeteredConnectionOnly`. Whilst most Wi-Fi connections are unmetered some are not (hotspot tethering, airport Wi-Fi), also with 5G there are some mobile connections which _are_ unmetered. 

My point being: we're presumably trying to avoid the user being charged for data rather than simply letting them choose a specific network protocol.

Reply:
Thanks for pointing that out. So does that mean I should update the label in the settings as well?

Reply:
Good catch, @dturner. Google Photos and other apps still prompt the user to "Wait for Wi-Fi" rather than "Wait for Unmetered connection". The latter is harder to explain to beginner users - @sergeydolgov1 do you think we should use "Wi-Fi" as a proxy for "Unmetered Network" in the UIs, even though it's not really 100% correct?

Reply:
@gino-m Suggest that we use "Wi-Fi" as a proxy for Unmetered Network as Wi-Fi is usually the way people use the Internet free of charge. Also, FYI -- this is Android developer documentation, they also use "Wi-Fi" terminology: https://developer.android.com/training/basics/network-ops/managing